BraveShared/Preferences.swift

```diff
@@ -124,7 +124,12 @@ extension Preferences {
         public static let seenDataMigrationFailureError = Option<Bool>(key: "rewards.seen-data-migration-failure-error", default: false)
         public static let migratedLegacyWallet = Option<Bool>(key: "rewards.migrated-legacy-wallet", default: false)
         public static let dismissedLegacyWalletTransfer = Option<Bool>(key: "rewards.dismissed-legacy-wallet-transfer", default: false)
+        public static let transferDrainID = Option<String?>(key: "rewards.legacy-wallet-transfer-drain-id", default: nil)
```

Comment:
Just a thought for future. 
What do you think about holding RewardsTransfer related  preferences in a whole `Transfer` object in core data?

Reply:
I guess we could? But this is just a one-off feature so its not super crucial. I could maybe create a `Transfer` category  inside of `Preferences.Rewards` if we wanted?

Reply:
yeah that can be a bit easier to track also with what is happening related with transfer info oyu know